source/renderer/app/i18n/locales/en-US.json

```diff
@@ -36,6 +38,12 @@
   "appUpdate.overlay.postponeInstall.link.label": "Postpone the update",
   "appUpdate.overlay.subtitle": "You are currently running Daedalus version <b>{currentAppVersion}</b>.<br />Daedalus version <b>{availableAppVersion}</b> is now available to download.",
   "appUpdate.overlay.title": "Software update available!",
+  "asset.transactionConfirmation.assetLabel": "!!!Token #{assetNumber}",
```

Comment:
There translations were removed automatically after I removed this unused [code](https://github.com/input-output-hk/daedalus/pull/2956/files#diff-d89b13bb59bd2d2a3412173a5c8d62a25607cb7672096154e28846d4c28d8d5fL98). Maybe they came back with those wrongly added wallet token file. We might need to run "manage:translations" again?

Reply:
Ok thanks, i will update to latest develop and manage translations 👍

Reply:
@lucas-barros but these translations are still used by components like https://github.com/input-output-hk/daedalus/blob/develop/source/renderer/app/components/assets/AssetTransactionConfirmation.tsx#L21 … i can see that there are no translations provided for them though … not sure if these components are still used somewhere? If not then we should probably rather delete them and not just the i18n keys (which are auto-generated)